spec/models/facilities_query_spec.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FacilitiesQuery do
+  describe '#query' do
+    let(:setup_pdx) do
+      %w[vc_0617V nca_907 vha_648 vha_648A4 vha_648GI vba_348
+         vba_348a vba_348d vba_348e vba_348h dod_001 dod_002].map { |id| create id }
+    end
+
+    it 'should find facility in the bbox' do
+      create :vha_648A4
+      bbox = ['-122.440689', '45.451913', '-122.786758', '45.64']
+      expect(FacilitiesQuery.new(bbox: bbox).query.first.id).to eq('648A4')
+    end
+
+    it 'should find facility by type' do
+      setup_pdx
```

Comment:
so with setup methods like this I'll often create a describe block with a before method
```
describe 'with existing pdx' do
  before(:each) do
    #the pdx setup stuff
  end
  it 'blah' ... etc
end
```

Reply:
What do you think about me putting that `setup_pdx` stuff in a before block, and moving that one test out of that `describe` context? This way we don't call `setup_pdx` in every one of those tests

Reply:
i think there's two tests that don't call `setup_pdx`.  That's a totally fine way to do it.  I prefer more and nested describe blocks because it puts more information in the test output, but pulling those two tests out of the `#query` describe block is fine too